.travis.yml

```diff
@@ -1,24 +1,33 @@
 language: python
-matrix:
-  include:
-    - python: 2.7
-      env: SYMPY_VERSION="oldest"
-    - python: 2.7
-      env: SYMPY_VERSION="latest"
-    - python: 2.7
-      env: SYMPY_VERSION="master"
-virtualenv:
-  system_site_packages: true
```

Comment:
Versions of python that are not 2.7 and 3.2 fail in Travis as site packages are not available. This shows an example of what happens if you leave the flag: https://github.com/travis-ci/travis-ci/issues/2219
which also happened with previous versions of this commit: https://travis-ci.org/pydy/pydy/pull_requests